sopt/src/main/scala/dagr/sopt/cmdline/ClpArgumentDefinitionPrinting.scala

```diff
@@ -98,7 +101,7 @@ object ClpArgumentDefinitionPrinting {
     val label = new StringBuilder()
     if (shortName.nonEmpty) label.append("-" + shortName + " " + shortType + ", ")
     label.append("--" + name + longType)
-    stringBuilder.append(wrapString(KGRN, label.toString(), KNRM))
+    stringBuilder.append(StringUtil.wrapString(KGRN, label.toString(), KNRM))
```

Comment:
Doh - when I'm writing snippets like this I still sometimes forget and use Java syntax :confounded: 
I did mean `case object` though, although I guess making them `case` objects doesn't really add much over being just `objects` in this case.